tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -583,17 +583,181 @@
   </ShaderOp>
 
   <ShaderOp Name="ComputeRawBufferLdStI32" CS="CS">
+    <RootSignature>RootFlags(0), SRV(t0), SRV(t1), UAV(u0), UAV(u1), DescriptorTable(SRV(t2,numDescriptors=2), UAV(u2,numDescriptors=2))</RootSignature>
+    <Resource Name="SRVBuffer0" Dimension="BUFFER" Width="160" InitialResourceState="COPY_DEST" Init="ByName" />
+    <Resource Name="SRVBuffer1" Dimension="BUFFER" Width="160" InitialResourceState="COPY_DEST" Init="ByName" />
+    <Resource Name="SRVBuffer2" Dimension="BUFFER" Width="160" InitialResourceState="COPY_DEST" Init="ByName" />
+    <Resource Name="SRVBuffer3" Dimension="BUFFER" Width="160" InitialResourceState="COPY_DEST" Init="ByName" />
+    <Resource Name="UAVBuffer0" Dimension="BUFFER" Width="480" Flags="ALLOW_UNORDERED_ACCESS" InitialResourceState="COPY_DEST" TransitionTo="UNORDERED_ACCESS" Init="ByName" Format="R32_SINT" ReadBack="true" />
```

Comment:
I think the format for all raw buffer resources should be `"R32_TYPELESS"`.  For StructuredBuffers, I think it's supposed to be `"UNKNOWN"`.  This goes for all the ShaderOp blocks.